### ISSUE TITLE
when executing a git command that needs GitHub authorization it fails

### ISSUE THREAD

**Issue by martinkooij** [GitCredentialManager/git-credential-manager#605]
[gcm-diagnose.log](https://github.com/GitCredentialManager/git-credential-manager/files/7835034/gcm-diagnose.log)
**Which version of GCM are you using?**

2.0.632+478753bbb7

**Which Git host provider are you trying to connect to?**

* [x] GitHub

**Can you access the remote repository directly in the browser using the remote URL?**

From a terminal, run `git remote -v` to see your remote URL.

* [x] Yes
* [ ] No, I get a permission error
* [ ] No, for a different reason - please describe

---

**Expected behavior**

I am authenticated and my Git operation completes successfully.

**Actual behavior**
executing a git push gives: 
```
fatal: Failed to open secret service session [0x13]
fatal: No such secret item at path: /org/freedesktop/secrets/collection/login/1
Username for 'https://github.com': 
```

**Logs**

Attached
```

**Comment by dscho** [comment]
I guess this is on Linux, given the path `/org/freedesktop/`? Can you share a bit more about your configuration?

**Comment by martinkooij** [comment]
yes: Linux Mint on an old laptop... OSType: Linux
OSVersion: Linux martin-mint 5.4.0-92-generic #103-Ubuntu SMP Fri Nov 26 16:13:00 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux.

Most information on my setup can be found in the attached log file at the top line of my original Post. I am a bit out of my wits because with seahorse I can see that the collection does not exist, but there are password collection in my home directory

**Comment by dscho** [comment]
Didn't you have to configure GCM to use `libsecret` specifically?

**Comment by dscho** [comment]
Also, since this seems to be a `libsecret`-related problem more than a GCM-related one, I allow myself to point you to https://unix.stackexchange.com/questions/265503/how-do-i-fix-no-such-secret-collection-at-path-for-gnome-keyring-and-arch-l as potentially containing hints helping you to resolve the problem.

**Comment by martinkooij** [comment]
@dscho : Indeed I configured with
````
git config --global credential.credentialStore secretservice
````
as per documentation https://github.com/GitCredentialManager/git-credential-manager/blob/main/docs/credstores.md

**Comment by dscho** [comment]
As per https://github.com/GitCredentialManager/git-credential-manager/blob/main/docs/credstores.md#freedesktoporg-secret-service-api, have you tried looking at things using `secret-tool` or `seahorse`?

**Comment by martinkooij** [comment]
yes seahorse gives an empty keyring where I would expect it to work. Also seahorse gives a warning. Mmmm..  I need to delve further into this.

**Comment by martinkooij** [comment]
Ok, I retried by unconfiguring secretservice and using pass. That fails as well, it seems to be somewhere else. Because yesterday it worked one time (I got the popup fro githun from secreteservice) and now nothing is working anymore, I suspect there is a deamon not running and git doesn't seem to recognize it should trigger gcm or one step later. However, running out of time now, will try one reboot, and if that fails, will pick up later this week. Thanks for reaching out!!
